jfrog_platform/README.md

```diff
@@ -0,0 +1,64 @@
+# Artifactory and Xray Logging Analytics with Datadog
+
+## Overview
+
+The following describes how to configure Datadog to gather metrics from Artifactory and Xray.
+
+## Requirements
+
+* Kubernetes Cluster
+* Artifactory and/or Xray installed via [JFrog Helm Charts](https://github.com/jfrog/charts)
+* Helm 3
+
+## Datadog Setup
+
+Datadog setup can be done by creating an account in datadog and going through onboarding steps or by using apiKey directly if one exists
+
+* Run the datadog agent in your kubernetes cluster by deploying it with a helm chart
+* To enable log collection, update datadog-values.yaml file given in the onboarding steps
```

Comment:
Just to clarify, what are the onboarding steps this bullet point is referring to? Maybe we can link the docs?

Reply:
We reworded to clarify these steps.